### PR TITLE
feat: CFD-444 export page design

### PR DESCRIPTION
**Pull request by Aaron-Collis-IW**
# Description

-   Explain the purpose of this pr

# Testing instructions

- auth against test
- run 
1. `awslocal s3api list-objects --bucket fdbt-netex-data-dev | jq ".Contents[].Key"`
2.`awslocal s3 cp s3://fdbt-netex-data-dev/BLAC/carnetFlatFare.xml s3://fdbt-netex-data-dev/BLAC/exports/TestExport/carnet.xml`
3. `awslocal s3 cp s3://fdbt-netex-data-dev/BLAC/carnetFlatFare.xml s3://fdbt-matching-data-dev/BLAC/exports/TestExport/carnet.xml`
- go to exports page and check it looks how it should on figma

# Type of change

-   [x] feat - A new feature
-   [ ] fix - A bug fix
-   [ ] docs - Documentation only changes
-   [ ] codestyle - Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc)
-   [ ] refactor - A code change that neither fixes a bug nor adds a feature
-   [ ] perf - A code change that improves performance
-   [ ] test - Adding missing tests or correcting existing tests
-   [ ] build - Changes that affect the build system or external dependencies (example scopes: gulp, broccoli, npm)
-   [ ] ci - Changes to our CI configuration files and scripts (example scopes: Travis, Circle, BrowserStack, SauceLabs)
-   [ ] chore - Other changes that don't modify src or test files
-   [ ] revert - Reverts a previous commit
-   [ ] content - Changes to content or copy

### CODE REVIEW

File: repos/fdbt-site/src/design/modules/_helpers.scss [hunk 1]
```diff
@@ -27,3 +27,7 @@
 .dft-text-align-centre {
     text-align: center;
 }
+
+.dft-four-fifths-width {
```
Comment (JwanKhalaf) [inline repos/fdbt-site/src/design/modules/_helpers.scss:31]: I don't think you need this, you can use the two-thirds. I think the design is just an oversight from Esmond.

File: repos/fdbt-site/src/pages/products/exports.tsx [hunk 2]
```diff
@@ -1,67 +1,106 @@
...
+                            flagged with ‘Needs Attention’ will be exported but may not be correct. Expired products or
+                            products for expired services, will not be included in the export. Once your export is
+                            complete, it will appear in the list below and you can download this and upload it to{' '}
+                            <a href="https://publish.bus-data.dft.gov.uk/">BODS</a>.
+                        </p>
                         <table className="govuk-table">
                             <thead className="govuk-table__head">
                                 <tr className="govuk-table__row">
                                     <th scope="col" className="govuk-table__header">
                                         Export Name
```
Comment (JwanKhalaf) [inline repos/fdbt-site/src/pages/products/exports.tsx:70]: Sentence casing on these.

File: repos/fdbt-site/src/pages/products/exports.tsx [hunk 3]
```diff
@@ -1,67 +1,106 @@
...
+                                    </th>
+                                    <th scope="col" className="govuk-table__header">
+                                        Export Time
+                                    </th>
+                                    <th scope="col" className="govuk-table__header">
+                                        Export Status
                                     </th>
                                     <th scope="col" className="govuk-table__header">
-                                        Status
+                                        Download File
```
Comment (JwanKhalaf) [inline repos/fdbt-site/src/pages/products/exports.tsx:82]: Not needed.

File: repos/fdbt-site/src/pages/products/exports.tsx [hunk 4]
```diff
@@ -1,67 +1,106 @@
...
+                            </CsrfForm>
+                        </div>
 
+                        <p className="govuk-body-m dft-four-fifths-width">
+                            This will export all of your current active products and any pending products. Any products
+                            flagged with ‘Needs Attention’ will be exported but may not be correct. Expired products or
+                            products for expired services, will not be included in the export. Once your export is
+                            complete, it will appear in the list below and you can download this and upload it to{' '}
+                            <a href="https://publish.bus-data.dft.gov.uk/">BODS</a>.
+                        </p>
```
Comment (JwanKhalaf) [inline repos/fdbt-site/src/pages/products/exports.tsx:65]: Some margin/spacing between this and the table.

File: repos/fdbt-site/src/pages/products/exports.tsx (hunk 3 above)
Comment (Aaron-Collis-IW) [reply repos/fdbt-site/src/pages/products/exports.tsx]: Done

File: repos/fdbt-site/src/design/modules/_helpers.scss (hunk 1 above)
Comment (Aaron-Collis-IW) [reply repos/fdbt-site/src/design/modules/_helpers.scss:31]: I think it looks better as four fifths as it fits better with the export button. Also what two thirds would you use? I can only find the column one which doesn't seem to work as expected

File: repos/fdbt-site/src/pages/products/exports.tsx (hunk 2 above)
Comment (Aaron-Collis-IW) [reply repos/fdbt-site/src/pages/products/exports.tsx]: Doen

File: repos/fdbt-site/src/pages/products/exports.tsx (hunk 4 above)
Comment (Aaron-Collis-IW) [reply repos/fdbt-site/src/pages/products/exports.tsx:65]: Can you help with this please?

**Review by JwanKhalaf — APPROVED** [review]
:ok_hand:

**Pull request merged**